gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeExtension.kt

```diff
@@ -31,6 +31,7 @@ abstract class ComposeExtension @Inject constructor(
      * ```
      * (see available versions here: https://developer.android.com/jetpack/androidx/releases/compose-kotlin#pre-release_kotlin_compatibility)
      */
+    @Deprecated("Since Kotlin $newCompilerIsAvailableVersion Compose Compiler configuration is moved to the \"$newComposeCompilerKotlinSupportPluginId\" plugin")
```

Comment:
As we still don't have compatible `org.jetbrains.kotlin.plugin.compose` plugin, we shouldn't deprecate it now.

Reply:
Maybe it's okay to keep this deprecation if we postpone merging this PR until kotlin 2.0.0-RC2.

Reply:
true